yarn-project/end-to-end/src/e2e_fees/private_refunds.test.ts

```diff
@@ -70,19 +74,27 @@ describe('e2e_fees/private_refunds', () => {
           ),
         },
       })
-      .wait();
+      .wait({ debug: true });
 
-    expect(tx.transactionFee).toBeGreaterThan(0);
+    expect(transactionFee).toBeGreaterThan(0);
+    // In total 4 notes should be inserted: 1 change note for user, 1 flat fee note for fee payer, 1 refund note for
+    // user and 1 fee note for fee payer.
+    expect(debugInfo?.noteHashes.length).toBe(4);
```

Comment:
Added the num notes check here since I wanted to use num notes in the following test and I needed to sanity-check that I am counting the notes correctly.